src/pages/RestaurantPageInfo/RestaurantPageInfo.scss

```diff
@@ -0,0 +1,19 @@
+.restaurant-picture {
+  width: 100%;
+  z-index: -1;
+}
+
+.restaurant-info-content {
+  background-color: white;
+  padding: 40px;
+  border-radius: 20px 20px 0 0;
+  border: 1px solid #BBBBBB;
+  position: relative;
+  top: -25px;
+
+  min-height: 100vh;
```

Comment:
what if we get rid of that `min-height` and let that container's content define it's size - just as we've discussed?

Reply:
I replaced min-height with an margin-bottom. Could you take a look at this?